app/models/run_history.rb

```diff
@@ -1,6 +1,15 @@
 class RunHistory < ApplicationRecord
   belongs_to :run
 
+  scope :completed, lambda { |timing|
+    case timing
+    when Run::REAL
+      where('realtime_duration_ms > 0').count
+    when Run::GAME
+      where('gametime_duration_ms > 0').count
+    end
+  }
```

Comment:
For multiline use `do ... end` rather than `{ ... }` to match convention.

Reply:
One of these years I'll get all the different conventions down XD

Reply:
Apparently there is some sort of precedence error when using a lambda in the scope function, so there has to be () around the lambda.  While looking to see what was the problem I saw that most people recommend that scopes be one or 2 lines, and everything else should be a function.  Should I leave it like this or change it around?

Reply:
I find that `rubocop` is a lifesaver for sticking to convention.

Reply:
The thing is rubocop is fine with either, but when I was google'ing around for the issue I saw other people saying that scopes should be short and sweet.

Reply:
Rubocop can't catch everything, we should stick with popular convention even if Rubocop has no opinion on it.